markdown_lab/network/async_client.py

```
@@ -0,0 +1,576 @@
+        for coro in asyncio.as_completed(tasks):
+            try:
+                url, result = await coro
+                results[url] = result
+                completed += 1
+
+                if progress_callback:
+                    progress_callback(completed, len(urls))
+
+                logger.debug(f"Completed {completed}/{len(urls)}: {url}")
+
+            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)